python/ray/data/dataset.py

```diff
@@ -3397,6 +3398,9 @@ def write_bigquery(
             dataset: The name of the dataset in the format of ``dataset_id.table_id``.
                 The dataset is created if it doesn't already exist. The table_id is
                 overwritten if it exists.
+            max_retry_cnt: The maximum number of retries that an individual block write
+                is retried due to BigQuery rate limiting errors. This isn't
+                related to Ray fault tolerance retries.
```

Comment:
Could you document what `max_retry_cnt=None` does?

Reply:
I'll go with your suggestion below and have `max_retry_cnt` be an int. I updated the docstring to include the default value.